pycm/pycm_obj.py

```diff
@@ -880,3 +880,65 @@ def combine(self, other):
         return ConfusionMatrix(
             matrix=matrix_combine(
                 self.matrix, other.matrix))
+
+    def plot(
+            self,
+            normalized=False,
+            one_vs_all=False,
+            class_name=None,
+            title='Confusion Matrix',
+            number_label=False,
+            cmap=None,
+            number_label_cmap=None,
+            plot_lib='matplotlib'):
+        """
+        Plot Confusion matrix.
+
+        :param normalized: normalized flag for matrix
+        :type normalized: bool
+        :param one_vs_all: one_vs_all flag for matrix
+        :type one_vs_all: bool
+        :param class_name: class name of one_vs_all action
+        :type class_name: bool
+        :param title: Plot title
+        :type title: string
+        :param number_label: number label flag
+        :type number_label: bool
+        :param cmap: color map
+        :type cmap: matplotlib.colors.ListedColormap
+        :param number_label_cmap: number label cmap
+        :type number_label_cmap: matplotlib.colors.ListedColormap
+        :param plot_lib: Plotting Library
+        :type plot_lib: string
+        :return: plot axes
+        """
+        matrix = self.to_array(
+            normalized=normalized,
+            one_vs_all=one_vs_all,
+            class_name=class_name)
+        if normalized:
+            title += " (Normalized)"
+        classes = sorted(self.classes)
+        if one_vs_all and class_name in classes:
+            classes = [class_name, '~']
+        ax = None
+        if plot_lib == 'matplotlib':
+            try:
+                from matplotlib import pyplot as plt
+            except ModuleNotFoundError:
+                raise pycmPlotError(MATPLOTLIB_PLOT_LIBRARY_ERROR)
+            fig, ax = plt.subplots()
+            if cmap is None:
+                cmap = plt.cm.gray_r
+            plt.imshow(matrix, cmap=cmap)
+            plt.colorbar()
+            fig.canvas.set_window_title(title)
+        elif plot_lib == 'seaborn':
+            try:
+                import seaborn as sns
+            except ModuleNotFoundError:
+                raise pycmPlotError(SEABORN_PLOT_LIBRARY_ERROR)
+            ax = sns.heatmap(matrix, cmap=cmap)
```

Comment:
Here, we have a problem in this case :
```pycon
>>> cm.plot(plot_lib = "seaborn",number_label=True)
```
**Error :** 
```
TypeError: 'NoneType' object is not callable
```
It's because,  `cmap` is `None`